src/script/index.js

```diff
@@ -2,20 +2,23 @@
 const path = require('path')
 const Promise = require('bluebird')
 const fs = Promise.promisifyAll(require('fs'))
-const { COMPONENT_RELATIVE_PATH } = require('./constants')
-const server = require('./server')
+const { COMPONENT_ABSOLUTE_PATH } = require('../constants')
+const server = require('../server')
 
 const start = async () => {
   // 1. Read the template file
   const filePath = path.resolve(__dirname, 'template.jsx')
   const template = await fs.readFileAsync(filePath, 'utf-8')
 
   // 2. Replace what needs to be replaced
-  const appFileContent = template.replace('/* react-workbench-insert import */', COMPONENT_RELATIVE_PATH)
+  const output = { dir: path.resolve(__dirname, '..', '..', 'tmp'), file: 'index.jsx' }
+  const appFileContent = template.replace(
+    '/* react-workbench-insert import */',
+    path.relative(output.dir, COMPONENT_ABSOLUTE_PATH)
```

Comment:
When my tested component is in the tmp folder I get `import Component from 'comp.jsx'` instead of `import Component from './comp.jsx'`, so it fails with webpack (due to absolute import)
I think it's not important for it's PR but we should create an issue maybe ?